app/models/profile.rb

```diff
@@ -50,6 +50,10 @@ def from_openscap_parser(op_profile, benchmark_id: nil, account_id: nil)
     end
   end
 
+  def policy_type
+    (parent_profile || self).name
```

Comment:
Observation: parent profile might want to be preloaded (`.includes(:parent_profile)`) in REST/GQL on index/collection queries.

Reply:
I was looking for the `N+1` query message but wasn't seeing it. It could be included during some other step, like while checking the profile is or isn't canonical. There are some `N+1` messages on certain endpoints, though, so we should track them down at some point as tech debt.